megamek/i18n/megamek/client/messages.properties

```diff
@@ -344,58 +352,49 @@ BoardView1.ROLLED=ROLLED
 BoardView1.sensorReturn=Sensor Return
 BoardView1.LowFoliage=FOLIAGE
 
-BoardView1.Tooltip.Unit=<B><FONT COLOR=#{0}>{1} ({2})</FONT></B>
-BoardView1.Tooltip.Pilot={0}: {1}
-BoardView1.Tooltip.PilotStart=<TABLE CELLSPACING=0 CELLPADDING=0><TBODY><TR><TD VALIGN=\"top\">
-BoardView1.Tooltip.PilotEnd=</TD></TR></TBODY></TABLE>
-BoardView1.Tooltip.PilotStatus=<FONT COLOR=#FF0000> ({0})</FONT>
-BoardView1.Tooltip.Advs=<BR><FONT SIZE=-2 COLOR=#555555>&nbsp;&nbsp;{0} Advs.;</FONT>
-BoardView1.Tooltip.Adv1=<BR><FONT SIZE=-2 COLOR=#555555>&nbsp;&nbsp;1 Adv.;</FONT>
-BoardView1.Tooltip.PilotPortraitW=</TR><TR><TD VALIGN=\"top\"><IMG WIDTH=60 HEIGHT={1} SRC=\"file:{0}\"></TD>
-BoardView1.Tooltip.PilotPortraitH=</TR><TR><TD VALIGN=\"top\"><IMG WIDTH={1} HEIGHT=60 SRC=\"file:{0}\"></TD>
+BoardView1.Tooltip.AeroVelAlt=<I>Velocity: {0}, Altitude: {1}</I>
+BoardView1.Tooltip.AeroStartingVelAlt=<I>Starting Velocity: {0}, Altitude: {1}</I>
 BoardView1.Tooltip.ArmorInternals=Armor {0}; Internal {1}
+BoardView1.Tooltip.ArmorCapital=<I>Capital Armor</I>
```

Comment:
Italics are terrible for accessibility, and should **never** be used. Use em or b instead.

Reply:
This holds for the whole file, as I see a number of other cases where this occurs.

Reply:
This has been like that before, lets maybe discuss outside. Easily changed.